remoulade/middleware/pipelines.py

```diff
@@ -37,20 +41,83 @@ def actor_options(self):
         }
 
     def after_process_message(self, broker, message, *, result=None, exception=None):
+        from ..composition import GroupInfo
+
+        if exception is not None:
+            return
+
+        pipe_target = message.options.get("pipe_target")
```

Comment:
do you think there should be a middleware for a group completion ? I cannot think of a good way to pass data beween midlewares. We could modify the message object but it seems bad, telle me what you think